paymentsheet/src/main/java/com/stripe/android/customersheet/CustomerSheetViewModel.kt

```diff
@@ -182,15 +181,15 @@ internal class CustomerSheetViewModel @Inject constructor(
 
     private fun onDismissed() {
         _result.update {
-            InternalCustomerSheetResult.Canceled
+            InternalCustomerSheetResult.Canceled(null)
```

Comment:
I expected this to be passed the currently selected PaymentOptionSelection, not always null. Is this a TODO, or did I misunderstand?